src/parachain/nomination.ts

```diff
@@ -188,22 +190,36 @@ export class DefaultNominationAPI extends DefaultTransactionAPI implements Nomin
         return nonceMap;
     }
 
-    async listNominatorsRaw<C extends CurrencyUnit>(
-        currency: Currency<C>
+    async listAllNominations(): Promise<
+        [[WrappedCurrency, [number, string, string]], MonetaryAmount<Currency<CollateralUnit>, CollateralUnit>][]
+        > {
+        let result: [
+            [WrappedCurrency, [number, string, string]],
+            MonetaryAmount<Currency<CollateralUnit>, CollateralUnit>
+        ][] = [];
+        for (const currency of WRAPPED_CURRENCIES) {
```

Comment:
Yeah, actually I prefer having the wrapped currency being set on the global API level rather than returning all the nominations combined for all wrapped currencies here.